.github/workflows/publish.yml

```diff
@@ -22,6 +22,54 @@ jobs:
       - name: Gulp
         run: npx gulp
         working-directory: ./Package
+      - name: Upload Assembled Folder
+        uses: actions/upload-artifact@v2
+        with:
+          name: 'Assembled'
+          path: Package/Assembled
+  build-windows:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2.3.3
+        with:
+          submodules: 'true'
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1.0.2
+      - name: Add nuget to PATH
+        uses: nuget/setup-nuget@v1
+        with:
+          nuget-version: '5.x'
```

Comment:
Same comments on naming as above, though I don't feel super strongly about it, so up to you.